unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDController.cs

```diff
@@ -555,6 +567,25 @@ private void InitializeExperiencesViewer(Transform currentViewTransform, Transfo
         view.ShowExperiencesButton();
     }
 
+    private void InitializeNotificationPanel(Transform currentPanelTransform, Transform previousPanelTransform)
+    {
+        if (currentPanelTransform == null)
+            return;
+
+        notificationViewerTransform = currentPanelTransform;
+        notificationViewerTransform.SetParent(view.leftWindowContainer, false);
+        notificationViewerTransform.SetAsLastSibling();
+        notificationViewerTransform.GetComponent<MainChatNotificationsComponentView>().OnClickedNotification += OpenClickedChat;
+    }
+
+    private void OpenClickedChat(string chatId)
+    { 
+        if(chatId == "#nearby")
+            OpenPublicChatChannel("#nearby", true);
```

Comment:
Lets store this `#nearby` string as a descriptive constant. This pattern is going to change in the channels feature (we will use `~nearby` instead of `#nearby` and it will make it easier to integrate when the moment arise.